mozci/push.py

```diff
@@ -1440,10 +1441,30 @@ def _map_failing_tasks(groups):
             backfill=_map_failing_tasks(failures_to_be_backfilled),
         )
 
+    def has_intermittent_bug(self, term: str, bugs: list[dict]) -> bool:
+        """
+        Identify whether an intermittent bug has be opened already based on a failure term, structured as below:
+        TEST-UNEXPECTED-FAIL | path/name.ext | subtest name and/or further information
+        """
+        values = term.split("|")
+        if len(values) != 3:
+            logger.warning(
+                f"Invalid failure term '{term}': considering the failure as new."
+            )
+            return False
+        _, full_path, info = values
+        # Consider that an intermittent bug exist for this issue based on its name + status
+        return any(
+            bug["status"] in ("NEW", "REOPENED")
+            and bug["summary"] == f"Intermittent {full_path} | single tracking bug"
```

Comment:
Please do an `ends_with` check instead without `Intermittent`. Sometimes these bugs get prefixed to indicate restrictions (e.g. `Intermittent Linux debug`, `Beta Intermittent`.